packages/cozy-client/src/associations/HasOne.js

```diff
@@ -0,0 +1,17 @@
+import get from 'lodash/get'
+import Association from './Association'
+
+export default class HasOne extends Association {
+  get raw() {
+    return this.target[this.name]
```

Comment:
Le `raw` designe les données brutes de la relation. Donc dans notre cas, ces donnés brutes sont plutot stockées dans `this.target.relationships[this.name]`

Reply:
Tu peux rajouter un cas de test d'ailleurs ?